node/src/Context.cpp

```diff
@@ -33,13 +33,15 @@ namespace node {
                         .ThrowAsJavaScriptException();
                     return;
                 }
-                std::string powerPreferenceOption = optionsObject.Get("powerPreference").ToString();
-                if (powerPreferenceOption == "default") {
-                    options.powerPreference = MLPowerPreference_Default;
-                } else if (powerPreferenceOption == "low-power") {
-                    options.powerPreference = MLPowerPreference_Low_power;
-                } else if (powerPreferenceOption == "high-performance") {
-                    options.powerPreference = MLPowerPreference_High_performance;
+                std::string powerPreference = optionsObject.Get("powerPreference").ToString();
+                transform(powerPreference.begin(), powerPreference.end(), powerPreference.begin(),
```

Comment:
Add header file `#include <algorithm>` for `std::transform` and #include <cctype>for std::tolower. Add `std` prefixes before `transform `and `tolower`

Reply:
Updated, PTAL,thanks! @fujunwei

Reply:
A better catch is to assert the usage

https://github.com/webmachinelearning/webnn-native/blob/main/src/webnn_native/openvino/GraphIE.cpp#L852
https://github.com/webmachinelearning/webnn-native/blob/main/src/webnn_native/dml/GraphDML.cpp#L444
```
const char* deviceName = devicePreference == ml::DevicePreference::Cpu ||
                                         devicePreference == ml::DevicePreference::Default
                                     ? "CPU"
                                     : "GPU";
```
or in creating context https://github.com/webmachinelearning/webnn-native/blob/main/src/webnn_native/WebnnNative.cpp#L73

Reply:
I will use switch and assert. Thanks! @fujunwei